ddtrace/contrib/rediscluster/patch.py

```diff
@@ -0,0 +1,51 @@
+# 3p
+import rediscluster
+import wrapt
+
+# project
+from ...pin import Pin
+from ...ext import redis as redisx
+from ...utils.wrappers import unwrap
+from ..redis.patch import traced_execute_command, traced_pipeline
+from ..redis.util import format_command_args
+
+
+def patch():
+    """Patch the instrumented methods
+    """
+    if getattr(rediscluster, '_datadog_patch', False):
+        return
+    setattr(rediscluster, '_datadog_patch', True)
+
+    _w = wrapt.wrap_function_wrapper
+    _w('rediscluster', 'StrictRedisCluster.execute_command', traced_execute_command)
+    _w('rediscluster', 'StrictRedisCluster.pipeline', traced_pipeline)
+    _w('rediscluster', 'StrictClusterPipeline.execute', traced_execute_pipeline)
+    Pin(service='redis', app='redis', app_type='db').onto(rediscluster.StrictRedisCluster)
+
+
+def unpatch():
+    if getattr(rediscluster, '_datadog_patch', False):
+        setattr(rediscluster, '_datadog_patch', False)
+        unwrap(rediscluster.StrictRedisCluster, 'execute_command')
+        unwrap(rediscluster.StrictRedisCluster, 'pipeline')
+        unwrap(rediscluster.StrictClusterPipeline, 'execute')
+
+
+#
+# tracing functions
+#
+
+def traced_execute_pipeline(func, instance, args, kwargs):
+    pin = Pin.get_from(instance)
+    if not pin or not pin.enabled():
+        return func(*args, **kwargs)
+
+    cmds = [format_command_args(c.args) for c in instance.command_stack]
```

Comment:
I wish I could reuse `redis.traced_execute_pipeline`, but rediscluster's `command_stack` is different.